lib/chef/config.rb

```diff
@@ -497,7 +497,8 @@ def self.path_accessible?(path)
     default(:syntax_check_cache_path) { cache_options[:path] }
 
     # Deprecated:
-    default(:cache_options) { { :path => PathHelper.join(file_cache_path, "checksums") } }
+    # Move this to the default value of syntax_cache_path when this is removed.
+    default(:cache_options) { { :path => PathHelper.join(config_dir, "syntaxcache") } }
```

Comment:
i can't tell if this also affects /var/chef/cache/checksums on the client as well or not?